### PR TITLE
Give the path to ocluster.cap through the command line

### PR DESCRIPTION
**Pull request by MisterDA**
and I've tried to make the readme and command-line parsing just a little bit nicer.

### CODE REVIEW

File: server/main.ml [hunk 1]
```diff
@@ -1,20 +1,33 @@
...
+let main debug workdir cap_file = Lwt_main.run (Server.main ~debug ~cap_file ~workdir)
 
-let term =
-  let ($) = Cmdliner.Term.($) in
-  Cmdliner.Term.pure main $
-  Cmdliner.Arg.(value & flag & info ["debug"]) $
-  Cmdliner.Arg.(required & pos 0 (some string) None & info ~docv:"WORKDIR" [])
+(* Command-line parsing *)
+
+open Cmdliner
```
Comment (kit-ty-kate) [inline server/main.ml:7]:
```suggestion
module Arg = Cmdliner.Arg
module Term = Cmdliner.Term
```
I prefer to have as little `open` as possible. I regard it as bad practice for myself.
I use it only for local opens (i try not to abuse them as well), for stdlibs and `Infix` submodules.

File: server/backend/check.ml [hunk 2]
```diff
@@ -392,9 +392,7 @@ let trigger_slack_webhooks ~stderr ~old_logdir ~new_logdir conf =
     | _ -> Lwt_io.write_line stderr ("Something when wrong with slack webhook "^Uri.to_string webhook)
   end
 
-let get_cap ~stderr =
-  let home = Sys.getenv "HOME" in
-  let cap_file = home^"/ocluster.cap" in (* TODO: fix that *)
+let get_cap ~stderr cap_file =
```
Comment (kit-ty-kate) [inline server/backend/check.ml:395]:
```suggestion
let get_cap ~stderr ~cap_file =
```
I prefer using labled parameters when dealing with non-abstract types (like string)

**Review by kit-ty-kate — CHANGES_REQUESTED** [review]
Looks good overall but there are just a couple of nitpicky things I prefer to avoid in my code.
Once those are addressed, this should be good to go.

File: server/main.ml (hunk 1 above)
Comment (MisterDA) [reply server/main.ml:7]: Ok, I spend to much time on the other ocurrent projects where that's the style.

File: server/backend/check.ml (hunk 2 above)
Comment (MisterDA) [reply server/backend/check.ml:395]: Ok, I've done that everywhere else and I forgot this one!

**Review by kit-ty-kate — APPROVED** [review]

**Comment by kit-ty-kate** [comment]
thx

**Pull request merged**